tests/Exception/Renderer/BlockNotFoundRendererTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\BlockBundle\Tests\Exception\Renderer;
+
+use PHPUnit\Framework\TestCase;
+use Sonata\BlockBundle\Block\BlockServiceManager;
+use Sonata\BlockBundle\Block\Service\BlockServiceInterface;
+use Sonata\BlockBundle\Exception\BlockNotFoundException;
+use Sonata\BlockBundle\Model\BlockInterface;
+use Symfony\Component\DependencyInjection\Container;
+
+/**
+ * @author Jérôme Engeln
+ */
+final class BlockNotFoundRendererTest extends TestCase
+{
+    public function testRenderWithBlockNotFoundException(): void
+    {
+        $this->expectException(BlockNotFoundException::class);
+
+        $service = $this->createMock(BlockServiceInterface::class);
+
+        $container = new Container();
+
+        $container->set('existing-block', $service);
+
+        $manager = new BlockServiceManager($container, []);
```

Comment:
You're testing the BlockServiceManager so the test should be in https://github.com/sonata-project/SonataBlockBundle/blob/4.x/tests/Block/BlockServiceManagerTest.php

I think this is already tested and you just need to update https://github.com/sonata-project/SonataBlockBundle/blob/4.x/tests/Block/BlockServiceManagerTest.php#L59-L61

Reply:
Oh I see, I'll do that, thank you